android/src/main/java/com/tailscale/ipn/ui/theme/Theme.kt

```diff
@@ -152,6 +152,18 @@ val ColorScheme.off: Color
 val ColorScheme.link: Color
   get() = onPrimaryContainer
 
+  val ColorScheme.exitOff: Color
```

Comment:
When surfaced within auto-completion, these names are unclear. Can we use more extended strings such as `exitNodeOffline`, `runAsExitNodeContainer`, etc.?

Reply:
@sonovawolf curious what your thoughts are here. I tried to give them fairly generic names in case we'd ever want to reuse them. Should we plan for these colors to be very specific, and name them accordingly?

Reply:
If you want to go totally generic you can use the [standard color aliases](https://login.tailscale.com/admin/design/colors) like `red-500`.

If you need a semantic one, I'd either use the Android ones like `surfaceContainer` or make a very specific one like `exitNodeOffline`. Android variables by default don't really have a concept of "intent" or "role" like many other design systems, but I defined some [here](https://www.figma.com/design/QgX0rndA763pwggR7ctHLh/Mobile-Apps?node-id=5981-1437&t=aQeNJ3KK5TnSh2mG-4). I'm not totally sure how many of them are in the code already.

For the card backgrounds I'd use `error` and `warning` respectively. For the text I'd use `onError` and `onWarning`, and for the button I'd just use custom colors since it's a one-off use case.

Reply:
Thanks! I'm going to go ahead and use the ones defined in the Figma so we can potentially reuse.